baip-0007.md

```diff
@@ -0,0 +1,58 @@
+BAIP: 0007
+Title: Proposal on bitASSETS reward and BTS destruction
+Authors: cn-vote   zhouxiaobao-2010
+Status: Accepted
+Type: Consensus
+Created: 2020-06-01
+
+# Abstract
+The purpose of this proposal is to establish a scientific financial model of BTS, make reasonable use of the commission income of the committee, open up resources and reduce expenditure, and promote the stable growth of BTS system value.
+
+# Specification
+
+### bitASSETS reward
+
+Intelligent assets are the original assets of the system, and the corresponding holders should enjoy the dividend of system growth.
+Reward amount: 20000 BTS per day
+Participating assets: bitCNY, bitUSD, bitEUR, bitRUBLE
+
+1.Reward conditions:
+1)The holding amount should exceed X. For example: those over 1000 yuan can participate in reward.
+2)The holding amount needs to last for a period of time . For example, 7 days. Rewards can only be distributed if the holding amount meets 1000bitcny or above for 7 consecutive days.
+
+2.Reward method:
+1)Participating accounts should register at the beginning of each cycle.Account does not participate in reward if not registered. The registration method can be to transfer any small number of bitassets to the specified account.
+2)The reward needs to be collected actively. If the reward is not collected within the time limit, this part of the reward will not be accumulated.The collection method can be to transfer any small number of bitasset to the specified account.
+3)If there are too few bitassets participating in the reward plan, set the upper limit of daily yield, such as 5 ‰.
+4)The more available bitassets you hold, the more rewards you get. Calculated as a minimum of 7 days. For example, if the number of bitCNY held within 7 days is 2000 3000 3000 1000 5000 5000 2000, then 1000 is the effective number. Note that the available bitassets do not include bitassets such as participating in Pending purchase\ compulsory liquidation and transfer.
+5)There is a reward for recommending new registered account to participate in lock in the reward plan. If the number of locked in bitassets in the account reaches a certain amount (e.g. 100000 bitcny, other bitassets are calculated based on the exchange rate year-on-year), it can be applied as a promotion account. If the new account recommended for registration by the account participates in the locked in of smart assets, the account can obtain corresponding rewards.
```

Comment:
Bad idea, what will stop me from registering additional accounts and hold bitassets there to gain additional rewards?

Reply:
Didn't fully understand your problem. Any account can apply for a lock in bonus.



This passage means that if you can introduce other new accounts to participate in the lock up bonus plan, you will get some extra rewards.



This is a propaganda measure.